apps/server/src/modules/school/domain/dto/slim-school.dto.ts

```diff
@@ -0,0 +1,17 @@
+import { SchoolPurpose } from '../type';
+
+// TODO: The prefix "Slim" is a suggestion to name objects that are a small partial of the actual object.
```

Comment:
Wofür wird es verwendet? Evtl. kann das anstatt Slim verwendet werden.

Reply:
Das wird momentan nur fürs Einladen von Lehrkräften anderer Schulen in Teams benutzt. 

Es wird dann noch einen weiteren Fall geben, wo die Schul-Liste für den Login rausgeht, die gerade noch über /schoolslist abgefragt wird. Ich hatte erst überlegt, dass man dafür nur einen Endpunkt macht. 

Man könnte natürlich auch zwei verschiedene Endpunkte sowie UCs dafür erstellen. Klingt auch nach einer sinnvollen Lösung für mich.

Ich frage mich nur prinzipiell: Wollen wir für jeden Use Case im Frontend auch einen Use Case im Backend? Oder mit anderen Worten: Wie genau interpretieren wir "Use Case"? Ich schätze, die Antwort liegt in der Sphäre von DDD und damit kenn ich mich einfach nicht so gut aus.

Reply:
Ich hab die DTOs jetzt Use Case spezifisch gemacht.